server/bad_actor.py

```diff
@@ -115,7 +115,10 @@ def _slackify_all(self) -> list:
                 "type": "section",
                 "fields": judgment_items,
             },
-            {
+        ]
+
+        if not self.bad_actor_api_response.auto_rejected:
```

Comment:
**nitpick:** This is totally a matter of opinion, but this is kind of a double negative right? "If not not accepted..." For readability (at least in my brain!) it'd make more sense to swap these.

Also for whatever reason, I guess I'd expect for the "else" to be the more default behavior like:
if auto_reject:
     do something special
else:
     do the common thing
     
As will all things, there could be a good reason to have the if/else in this order that I'm not thinking of so open to your thoughts!

Reply:
No, I think that's a good point. In my head I still wanted the actions block to be handled first, but I do think this reads better and there's really no good reason to do it the other way.